docs/family-pub-key.rst

```diff
@@ -80,7 +80,17 @@ Inputs and Outputs
 
 The inputs and outputs for Pub Key family transactions in respect to payload must include:
 
-* **NewPubKeyPayload**: Sender's account address, *public_key address*
+* **NewPubKeyPayload**:
+  * Inputs:
+    * Sender's account address
+    * *public_key address*
+    * Address of setting *remme.economy_enabled*
+    * Address of setting *remme.settings.storage_pub_key*
+    * Address formed from value of setting *remme.settings.storage_pub_key*
```

Comment:
Why we need this address?

Reply:
For now we send tokens from commissions to this address.